src/script/connection/ConnectionRepository.ts

```
@@ -229,7 +234,6 @@ export class ConnectionRepository {
             break;
           }
         }
-        return false;
       }
       throw error;
```

Instead of returning false, we just throw an error we will ignore later

---

humm this actually changes the behavior. Previously we would catch unexpected errors, but now they are going to be swallowed into the void

---

If you follow the path starting from `UserActions.tsx`, we were actually never catching any (unexpected) errors seems like.

---

fixed, by returning `null` in this case